ethexe/service/src/tests.rs

```diff
@@ -1090,6 +1151,7 @@ mod utils {
         }
     }
 
+    // TODO (breathx): remove me in favor of crate::config::NodeConfig.
```

Comment:
I do not think this should be done. This config is special for testing purposes. It must be possible to simply expanded by developer for testing.